src/standalone/Snow/Snow.jl

```diff
@@ -30,13 +30,13 @@ struct SnowParameters{FT <: AbstractFloat, PSE}
     "Thermal conductivity of ice (W/m/K)"
     κ_ice::FT
     "Timestep of the model"
-    Δt::FT
+    Δt::Any
```

Comment:
do we need to even pass in a type if it is Any? What does it gain us?

Reply:
We don't, but for some reason the formatter keeps adding this... I'll try to remove it again

Reply:
Yeah, the formatter just adds it back if I try to remove it :/

Reply:
hahah, the formatter is being creative